engines/wintermute/base/gfx/opengl/base_surface_opengl3d.cpp

```diff
@@ -27,8 +27,49 @@
 
 namespace Wintermute {
 
+template<class T>
```

Comment:
Instead of copying this from the Grim-engine, I'd suggest moving it into math/, and use that version in both places. Finding a power-of-two suitable for texture dimensions is a problem common to most of our engines' GL renderers anyhow.

Reply:
Actually there is a nextHigher2 function in algorithm.h. I will use that one